src/cljs/main/broadfcui/common.cljs

```diff
@@ -179,16 +179,17 @@
 
 ; The list of all access levels in the system, inclusive of roles that aren't directly assignable
 ; Note that if you add an access level, you will want to add it in acl_editor.cljs as well
-(def ^:private access-levels-ascending ["NO ACCESS", "READER", "WRITER", "OWNER", "PROJECT_OWNER"])
+(def ^:private access-levels
+  {"NO ACCESS" 0 "READER" 1 "WRITER" 2 "OWNER" 3 "PROJECT_OWNER" 4})
 
-(defn access-greater-than? [this that]
-  (> (.indexOf (to-array access-levels-ascending) this) (.indexOf (to-array access-levels-ascending) that)))
+(defn access-greater-than? [level1 level2]
```

Comment:
These 3 functions now seem trivial enough to inline (but are also okay to keep as is).